calpip/Task1/Task 1-2.txt

```diff
@@ -1,24 +1,24 @@
 Dear Sir or Madam,
 
 A couple of months ago I bought a long term membership card to one of 
-your gyms located in Winnipeg downtown area. I have paid a considerable 
-high amount of money expecting the best quality of service in return. 
+your gyms located in Winnipeg downtown area. I have paid a considerably 
+high amount of money, expecting the best quality of service in return. 
 Sadly, my experience has been really bad so far.
 
-For one thing, the main reason why I have become a customer was the 
-variety of fitness class announced in your flyers. However, most of 
+For one, the main reason why I became a customer was for the 
+variety of fitness classes announced in your flyers. However, most of 
 those classes, for instance spinning and yoga, simply don’t exist in 
 the gym schedule. In addition, the boxing instructor is absent most of 
-the time and the gym doesn’t provide any replacement for that.
+the time and the gym doesn’t provide a replacement instructor.
 
-For another thing, I have spoken about these problems with the manager 
+I have also spoken about these problems with the manager 
```

Comment:
`For another thing, ` should be `Another thing, `. I didn't use that phrase again because you already used it earlier. So, I changed it to `I have also` for the same effect and without sounding repetitive.